src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -82,4 +82,20 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+
+    /**
+     * Get a printable string consists of the given printables
+     * @param printables
```

Comment:
Normally, you should give a description of the `@param` or the `@return` value. If you have nothing more to add, probably the comment line can be deleted as well (because it's not adding any value)?